app/models/user.rb

```diff
@@ -1,12 +1,38 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_devise.md#:~:text=Ask%20for%20confirmation%20of%20email.), Please add `:confirmable` module here to make the app ask for the email confirmation.
[Hint] See this [video](https://www.youtube.com/watch?v=nun18zM8Gq0) for more info.